internal/mailer/template.go

```diff
@@ -245,7 +245,7 @@ func (m *TemplateMailer) RecoveryMail(user *models.User, otp, referrerURL string
 
 	return m.Mailer.Mail(
 		user.GetEmail(),
-		string(withDefault(m.Config.Mailer.Subjects.Recovery, "Reset Your Password")),
+		withDefault(m.Config.Mailer.Subjects.Recovery, "Reset Your Password"),
```

Comment:
Minor refactor:  `string(` cast doesn't seem to be needed as `withDefault` already returns a string

Reply:
no need to cherry-pick, looks good!